tests/molecules/C4_ae_MSD_excited/CMakeLists.txt

```diff
@@ -0,0 +1,235 @@
+IF (NOT QMC_CUDA)
+ IF (NOT QMC_COMPLEX)
+
+#Nb_Det = 1
+# -150.95531020041986
+
+#  Reference from VMC runs
+  LIST(APPEND C4_SD_AE_NO_SHORT "totenergy" "-150.95310200 0.07367645")
```

Comment:
Same issue, 955 or 953?

Reply:
Fixed the reference to QP so we avoid bleedings from density fitting